### PR TITLE
README improvements and better error handling

### LINKED ISSUE #255: Running against old ratbagd throws an exception
Because of course it does, the GetSvgFd method isn't available. This becomes an issue when users rely on the distribution ratbagd (possibly outdated) and install piper from flathub (not outdated).

While staring at a mouse trap for hours on end is an improvement over the average TV program, it's probably not what users intended when they started piper.

### PR DESCRIPTION
**Pull request by whot**
Fixes #255,  #252, #254, #238

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,23 @@
...
 
-Piper is a GTK+ application to configure gaming mice, using libratbag via
-ratbagd.
-
-In order to run Piper, `ratbagd` has to be running (without it, you'll get to
-see a pretty mouse trap). To see how, see [its
+Piper is a GTK+ application to configure gaming mice. Piper is merely a
+graphical frontend to the ratbagd DBus daemon, see [the libratbag
 README](https://github.com/libratbag/libratbag/blob/master/README.md#running-ratbagd-as-dbus-activated-systemd-service).
+for instructions on how to run ratbagd.
```
Comment (Hjdskes) [inline README.md]: Should remove the period before this part.

File: README.md [hunk 2]
```diff
@@ -20,9 +27,26 @@ Screenshots
...
+either ratbagd is not running (like in this screenshot), ratbagd needs to be
+updated to a newer version, or some other unexpected error occured.
+
+![The error page](https://github.com/libratbag/piper/blob/wiki/screenshots/piper-errorpage.png)
+
 Installing Piper
 ================
 
+Most popular distributions package Piper and it is available through the
+packaging system (apt, dnf, yum, pacman, ...).
```
Comment (Hjdskes) [inline README.md]: Maybe add "This should be the your preferred way of getting Piper".

File: README.md [hunk 3]
```diff
@@ -20,9 +27,26 @@ Screenshots
...
+
+Piper is also available as a
+[Flatpak]([200~https://flathub.org/apps/details/org.freedesktop.Piper). When
```
Comment (Hjdskes) [inline README.md]: Remove "When"

**Comment by whot** [comment]
all fixed as requested

File: README.md [hunk 4]
```diff
@@ -23,6 +21,18 @@ Screenshots
 Installing Piper
 ================
 
+Most popular distributions package Piper and it is available through the
+packaging system (apt, dnf, yum, pacman, ...). This is the preferred way of
+installing Piper.
+
+Piper is also available as a
+[Flatpak]([200~https://flathub.org/apps/details/org.freedesktop.Piper).
```
Comment (Hjdskes) [inline README.md]: Just noticed that this link isn't showing in the rendered README.. sorry

Comment (whot) [reply README.md]: doh, should've checked. fixed now

**Pull request merged**